### DISCUSSION TITLE
gatsby-plugin-google-tagmanager: Variable base url for gtm container - Interested in a PR?

### DISCUSSION THREAD

**Discussion by tackaberry** [gatsbyjs/gatsby#32719] [Ideas / Feature Requests]
With the advent of Server Side Tagging, we have the ability to serve `gtm.js` from "self-hosted" tagging server. 
See: https://developers.google.com/tag-manager/serverside
And: https://developers.google.com/tag-manager/serverside/send-data#update_the_gtmjs_source_domain

Thus the base url, instead of `googletagmanager.com` could be `sst.example.com` or `analytics.example.com`. 

We have a working copy of the plugin that enables this.  Are you interested in a PR to support this on the official package?

**Comment by wardpeet** [comment]
Definitely, how would you call the option? selfHostedOrigin?

**Comment by tackaberry** [comment]
Sure.  I'll start with that.  I had `baseUrl`.  I'll tag you in the PR.

Brett Tackaberry

**Comment by tackaberry** [comment]
@wardpeet here is the PR: https://github.com/gatsbyjs/gatsby/pull/32733
